content/en/docs/kubernetes/operator/automatic.md

```diff
@@ -180,6 +180,67 @@ spec:
 
 For more details, see [.NET Auto Instrumentation docs](/docs/zero-code/net/).
 
+### Deno
+
+The following command creates a basic Instrumentation resource that is
+configured for instrumenting Deno services.
+
+```bash
+kubectl apply -f - <<EOF
+apiVersion: opentelemetry.io/v1alpha1
+kind: Instrumentation
+metadata:
+  name: demo-instrumentation
+spec:
+  env:
+    - name: OTEL_DENO
+      value: 'true'
+  exporter:
+    endpoint: http://demo-collector:4318
+  propagators:
+    - tracecontext
+    - baggage
+  sampler:
+    type: parentbased_traceidratio
+    argument: '1'
+EOF
+```
+
+Deno processes automatically export telemetry data to the configured endpoint
+when they are started with the `OTEL_DENO=true` environment variable. Therefore,
+the example specifies this environment variable in the `env` field of the
+Instrumentation resource, so it is set for all services that have env vars
+injected with this Instrumentation resource.
+
+By default, the Instrumentation resource that auto-instruments Deno services
+uses `otlp` with the `http/proto` protocol. This means that the configured
+endpoint must be able to receive OTLP over `http/proto`. Therefore, the example
+uses `http://demo-collector:4318`, which connects to the `http/proto` port of
+the `otlpreceiver` of the Collector created in the previous step.
+
+{{% alert title="Note" color="info" %}}
+
+Deno's OpenTelemetry integration is not yet stable. As a result all workloads
+that want to be instrumented with Deno must have the `--unstable-otel` flag set
+when starting the Deno process.
+
+{{% /alert %}}
+
+#### Configuring console.log capturing in Deno
+
+By default, the Deno OpenTelemetry integration captures `console.log` output as
+logs, while still printing the logs to stdout / stderr. There are two other
+behaviours that can be configured:
+
+- `OTEL_DENO_CONSOLE=replace`: capture `console.log` output as logs and do not
+  print to stdout / stderr.
+- `OTEL_DENO_CONSOLE=ignore`: do not capture `console.log` output as logs.
```

Comment:
```suggestion
- `OTEL_DENO_CONSOLE=ignore`: do not export `console.log()` output as logs.
```

Question: does `ignore` imply that the output is completely ignored, and hence output isn't printed either? If so then the statement should probably be:

```suggestion
- `OTEL_DENO_CONSOLE=ignore`: neither export as logs nor print `console.log()` output.
```

Reply:
Updated and clarified the behaviour.